src/mod_authn_file.c

```diff
@@ -646,6 +654,7 @@ static handler_t mod_authn_file_htpasswd_basic(server *srv, connection *con, voi
             #endif
            #endif
            #ifdef USE_OPENSSL_CRYPTO /* (for MD4_*() (e.g. MD4_Update())) */
```

Comment:
Please add a comment that says that wolfSSL defines this if MD4 is not provided, e.g.
```
#ifndef NO_MD4 /*(defined if support for MD4 is not built into wolfSSL)*/
```

Reply:
MD4 is disabled in wolfSSL by default. However, MD4 is an available wolfSSL configure option, should the user desire it.